cms/djangoapps/contentstore/views/item.py

```diff
@@ -751,6 +762,37 @@ def _get_module_info(xblock, rewrite_static_links=True, include_ancestor_info=Fa
         return xblock_info
 
 
+def _get_gating_info(course, xblock):
+    """
+    Returns a dict containing gating information for the given xblock which
+    can be added to xblock info responses.
+
+    Arguments:
+        course (CourseDescriptor): The course
+        xblock (XBlock): The xblock
+
+    Returns:
+        dict: Gating information
+    """
+    info = {}
+    if xblock.category == 'sequential':
+        # Gating info
+        if course.enable_subsection_gating:
```

Comment:
line 498, the same check is written in a different way:  if xblock.location.category == 'sequential' and course.enable_subsection_gating

xblock.location.category vs xblock.category, and 2-lines vs single line. Perhaps you can choose one over the other for both places?